translations/general/application/en-ca.json

```diff
@@ -1,7 +1,7 @@
 {
   "general": {
     "application": {
-      "name": "ember-boilerplate"
+      "name": "Ember Boilerplate"
```

Comment:
Wasn't this name like that to be auto-replaced by the script when starting a project from a boilerplate?

Reply:
You’re right, I forgot about that. I changed it on master! Thanks!